lib/media.js

```diff
@@ -182,8 +183,9 @@ function getStyleFiles(state) {
     assetDir = site && site.assetDir || MEDIA_DIRECTORY,
     assetPath = site && site.assetPath || '',
     assetHost = site && site.assetHost || '',
-    siteStyleguide = site && site.styleguide;
-  let cssFilePaths = [];
+    siteStyleguide = site && site.styleguide,
+    layoutName = _.get(state, '_layout.name');
```

Comment:
`&&` and `_.get` are doing the same thing here, right? I think within the same file we should aim for consistency

Reply:
good point